app/components/wallet-overview/wallet-overview-redesigned.tsx

```diff
@@ -0,0 +1,214 @@
+import React, { useEffect, useState } from "react"
+import ContentLoader, { Rect } from "react-content-loader/native"
+import { Pressable, StyleProp, TouchableHighlight, View, ViewStyle } from "react-native"
+
+import { gql } from "@apollo/client"
+import {
+  useHideBalanceQuery,
+  useWalletOverviewScreenQuery,
+  WalletCurrency,
+} from "@app/graphql/generated"
+import { useIsAuthed } from "@app/graphql/is-authed-context"
+import { useDisplayCurrency } from "@app/hooks/use-display-currency"
+import { toBtcMoneyAmount, toUsdMoneyAmount } from "@app/types/amounts"
+// import { testProps } from "@app/utils/testProps"
+import { makeStyles, Text, useTheme } from "@rneui/themed"
+
+import { GaloyCurrencyBubble } from "../atomic/galoy-currency-bubble"
+import { GaloyIconButton } from "../atomic/galoy-icon-button"
+import { GaloyIcon } from "../atomic/galoy-icon"
+
+type HidableAreaProps = {
+  hidden: boolean
+  style?: StyleProp<ViewStyle>
+  children: React.ReactNode
+}
+
+const Loader = ({ styles }: { styles: StyleProp<ViewStyle> }) => (
+  <View style={styles}>
+    <ContentLoader
+      height={"20%"}
+      width={"80%"}
+      speed={1.2}
+      backgroundColor="#f3f3f3"
+      foregroundColor="#ecebeb"
+    >
+      <Rect x="0" y="0" rx="4" ry="4" width="100%" height="100%" />
+    </ContentLoader>
+  </View>
+)
+
+const HidableArea = ({ hidden, style, children }: HidableAreaProps) => {
+  return (
+    <TouchableHighlight style={style} underlayColor="#ffffff00">
```

Comment:
Does this need to be a`TouchableHighlight`? There is no action being passed in for `onPress`.